src/ng-generate/table/index.ts

```diff
@@ -68,6 +68,10 @@ export function generateTable(options: Schema): Rule {
         options.skipImport = false;
     }
 
+    if(options.configFile !== WIZARD_CONFIG_FILE) {
+        options.configFile = WIZARD_CONFIG_FILE;
+    }
+
```

Comment:
Looking at the fact that we have 2 if statements here with setting default options I would suggest to update slightly the code above
```
const defaultOptions = {
  skipImport: false,
  configFile: WIZARD_CONFIG_FILE
};

options = {
  ...defaultOptions,
  ...oprions
};
```
The `defaultOptions` constant may or should be placed at a separate file